pom.xml

```diff
@@ -727,6 +727,11 @@
         <artifactId>microprofile-health</artifactId>
         <version>2.3.0.Final-SNAPSHOT</version>
       </dependency>
+      <dependency>
+        <groupId>io.thorntail</groupId>
+        <artifactId>microprofile-health-api</artifactId>
```

Comment:
I don't think an artifact like this exists.

Reply:
You were very polite. Sorry for being so clumsy.